tests/GenerateScriptRegressionTests/CompareDirs.cs

```diff
@@ -0,0 +1,86 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Xunit.Abstractions;
+
+namespace GenerateScriptRegressionTests;
+
+internal class CompareDirs
+{
+    public static bool Compare(string pathA, string pathB, ITestOutputHelper outputHelper)
+    {
+
+        DirectoryInfo dir1 = new DirectoryInfo(pathA);
+        DirectoryInfo dir2 = new DirectoryInfo(pathB);
+
+        List<FileInfo> list1 = dir1.GetFiles("*.*", SearchOption.AllDirectories).ToList();
+        List<FileInfo> list2 = dir2.GetFiles("*.*", SearchOption.AllDirectories).ToList();
+
+        FileCompare myFileCompare = new FileCompare();
```

Comment:
The use of multiple things called FileCompare (both a class and a method) in this method are confusing to me.

Reply:
Changed the class name to FileInfoComparer